UnityProject/Assets/Scripts/Health/Sickness/Sickness.cs

```diff
@@ -7,6 +7,7 @@
 using System.Linq;
 using Chemistry;
 using UnityEngine;
+using UnityEngine.ResourceManagement.ResourceProviders.Simulation;
```

Comment:
what's this namespace?

Reply:
probably some rider accidental autocompletion left over, removing.